Changelog

```diff
@@ -1,3 +1,60 @@
+Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.6.0
+    - FEATURE: Add (experimental) I2P support (glassez) cdded6cef7710fb25093ea142edea3d3b7479447
+    - FEATURE: Provide UI Theme editor (glassez) 77aa85fbd3af12b319f8f0b8372f4ba186d10c80
+    - FEATURE: Various UI theming improvements (glassez)
+    - FEATURE: Implement torrent tags editing dialog (glassez) d40be79c6933fd911644b0b0c833893ea480dc90
+    - FEATURE: Implement "Add torrent params" editing widget (glassez) 0cc29f18510b084b131fd0b02963a7aff909de01
```

Comment:
This can be considered as internal of the following feature: Revamp "Watched folder options" and "Automated RSS downloader" dialog